### DISCUSSION TITLE
Portable version?

### DISCUSSION THREAD

**Discussion by addison74** [dnGrep/dnGrep#1029] [Q&A] [answered]
I am using Everything search but it seems dnGrep offers other features. Is there any portable version? I don't want to install it.

**Comment by doug24** [comment, marked as answer]
Yes, just choose one of the zip files in the [releases](https://github.com/dnGrep/dnGrep/releases/tag/v4.1.7.0).

Unzip it anywhere and it will run without installing.  If you want to put a shortcut to dnGrep on the (old) File Explorer context menu, you can do that from the dnGrep Options dialog when running dnGrep as Administrator and it will add the registry entries for the menu.  Otherwise nothing gets written to the registry or stored outside of the extracted directory.
